testdata/go/v4-alpha/monitoring/memcached-operator/go.mod

```diff
@@ -3,13 +3,14 @@ module github.com/example/memcached-operator
 go 1.19
 
 require (
-	github.com/onsi/ginkgo/v2 v2.1.4
-	github.com/onsi/gomega v1.19.0
+	github.com/onsi/ginkgo/v2 v2.1.6
+	github.com/onsi/gomega v1.20.1
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.61.1
 	github.com/prometheus/client_golang v1.12.2
-	k8s.io/api v0.25.0
-	k8s.io/apimachinery v0.25.0
-	k8s.io/client-go v0.25.0
-	sigs.k8s.io/controller-runtime v0.13.0
+	k8s.io/api v0.25.4
```

Comment:
Can we avoid bumping k8s version in this PR? This is because the k8s bumps (minor or major) are usually tracked separately so as changes happening due to them.

Reply:
I'm not sure. Maybe it's running `go mod tidy` after generating? How can I avoid these bumps?